components/dashboards-web-component/src/listing/DashboardListing.jsx

```diff
@@ -1,121 +0,0 @@
-/*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This file is not deleted, rather it is renamed into `DashboardListingPage`. See commit 56ad93ee3ec03b0f0c71ca3e0817fd0e3a56d681.